rdkit/Chem/Pharm2D/SigFactory.py

```diff
@@ -92,7 +95,7 @@ def GetBitDescriptionAsText(self, bitIdx, includeBins=0, fullPage=1):
       a string with the HTML
 
     """
-    nPts, combo, scaffold, labels, dMat = self._GetBitSummaryData(bitIdx)
+    raise NotImplementedError('Missing implementation')
```

Comment:
nice one. This must have been a leftover from a (much) older version of the code